src/de/jost_net/JVerein/gui/control/MitgliedskontoControl.java

```diff
@@ -608,6 +611,34 @@ private void refreshMitgliedkonto2() throws RemoteException
     mitgliedskontoList2.sort();
   }
 
+  public Part getBuchungenList() throws RemoteException
+  {
+    if (buchungList != null)
+    {
+      return buchungList;
+    }
+    DBIterator<SollbuchungPosition> sps = Einstellungen.getDBService().createList(SollbuchungPosition.class);
+    sps.addFilter( "sollbuchung = ?", getMitgliedskonto().getID());
+    
+    buchungList = new TablePart(sps, null);
+    buchungList.addColumn("Datum", "datum",
+        new DateFormatter(new JVDateFormatTTMMJJJJ()));
+    buchungList.addColumn("Zweck", "zweck");
+    buchungList.addColumn("Betrag", "betrag",
+        new CurrencyFormatter("", Einstellungen.DECIMALFORMAT));
+    buchungList.addColumn("Steuersatz", "steuersatz");
+    buchungList.addColumn("Buchungsart", "buchungsart");
+    if (Einstellungen.getEinstellung().getBuchungsklasseInBuchung())
```

Comment:
Hier könnte man den BuchungsartFormatter und den BuchungsklasseFormatter setzen. Damit erfolgt die Darstellung wie in den Einstellungen konfiguriert.

Reply:
habe ich hinzugefügt